src/test/functional/pages/BuzzPage.ts

```diff
@@ -1,6 +1,44 @@
 import { BasePage } from "./BasePage";
-
+import {  expect, Locator, Page } from "@playwright/test";
 export class BuzzPage extends BasePage {
```

Comment:
rozszerzasz BasePage, ale nie widzę, byś korzystał tu z BasePage

Reply:
tutaj korzystam await buzzPage.navigateToSubPage('span:has-text("Buzz")');

Reply:
nie tworzy sie obiektu basePage po to robimy dziedziczenie zeby mozna byla instancja klasy ktora dziedziczy po basePage odwolac sie do funkcji z basePage